ArduCopter/Copter.cpp

```diff
@@ -182,7 +182,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(check_dynamic_flight,  50,     75,  72),
 #endif
 #if LOGGING_ENABLED == ENABLED
-    SCHED_TASK(fourhundred_hz_logging,400,    50,  75),
+    SCHED_TASK(loop_rate_logging, LOOP_RATE,    50,  75),
```

Comment:
Why not `FAST_TASK(loop_rate_logging)`?

Reply:
because then its not scheduled and cannot be delayed. Ideally everything should be in the scheduler because then it can be skipped under load if necessary

Reply:
So we should probably move `Log_Video_Stabilisation` to this style. It could even go in this `loop_rate_logging` function added here.